conversion-measurement/publisher/views/article.pug

```diff
@@ -16,22 +16,23 @@ block content
         const adScriptUrl = !{JSON.stringify(adScriptUrl)};
         let eventType = ""
         if (adScriptUrl.includes("view")) {
-          textStart = `A source event for this ad view has been registered in your browser`
+          textStart = `<hint> \n A source event for this ad view has been registered in your browser.\n`
         } else if (adScriptUrl.includes("click")) {
-          textStart = `Click on the ad. Once you do, a source event for this ad will be registered in your browser`
+          textStart = `<hint> \n Click on the ad. \n \n Once you do, a source event for this ad will be registered in your browser.`
```

Comment:
```suggestion
          textStart = `💡 Hint: \n Click on the ad. \n \n Once you do, a source event for this ad will be registered in your browser.`
```
or
```suggestion
          textStart = `💡 Demo tip: \n Click on the ad. \n \n Once you do, a source event for this ad will be registered in your browser.`
```

Or any other phrasing, but without `<>`. 
I'm inclined to avoid the tag-like notation, as the toast only supports text, so it may look like a rendering issue.
WDYT?

Reply:
Makes sense, happy with those changes.